test/lib/dashboard-agent.spec.js

```diff
@@ -64,18 +66,38 @@ describe("dashboard-agent", function () {
 
       var checkMetrics = function (metrics) {
         expect(metrics).to.be.an("object");
-        expect(metrics.eventLoop).to.deep.equal({ delay: 0, high: 0 });
-        expect(metrics.mem.systemTotal).to.be.above(0);
-        expect(metrics.mem.rss).to.be.above(0);
-        expect(metrics.mem.heapTotal).to.be.above(0);
-        expect(metrics.mem.heapUsed).to.be.above(0);
-        expect(metrics.cpu.utilization).to.be.above(0);
+        expect(metrics.eventLoop.delay).to.be.a("number");
+        expect(metrics.eventLoop.high).to.be.a("number");
+        expect(metrics.mem.systemTotal).to.equal(20);
+        expect(metrics.mem.rss).to.equal(30);
+        expect(metrics.mem.heapTotal).to.equal(40);
+        expect(metrics.mem.heapUsed).to.equal(50);
+        expect(metrics.cpu.utilization).to.equal(60);
       };
 
+      var stubProcess = sinon.stub(process, "memoryUsage", function () {
```

Comment:
One thing I did notice was the sandbox create was in a `before()` (pre-existing code). This may be problematic, and as a general rule for unit tests, `beforeEach()` is almost always better than a `before()`. `before()` is typically reserved for super-expensive time-wise things, and I don't see that here. (Same for `after()`)